docs/self-managed/setup/deploy/openshift/redhat-openshift.md

```diff
@@ -4,64 +4,106 @@ title: "Red Hat OpenShift"
+- Any version not listed below hasn't been tested, and we do not guarantee compatibility.
+- The OpenShift end of support date must be in the future: we do not ensure compatibility with versions no longer supported by Red Hat, as indicated in the "End of Support Date" column.
 
-#### Permissive SCCs
+| OpenShift Version | [End of Support Date](https://access.redhat.com/support/policy/updates/openshift) |
+| ----------------- | --------------------------------------------------------------------------------- |
+| 4.15.x            | August 27, 2025                                                                   |
+| 4.14.x            | May 1, 2025                                                                       |
+| 4.13.x            | November 17, 2024                                                                 |
+| 4.12.x            | July 17, 2024                                                                     |
 
-Out of the box, if you deploy Camunda 8 (and related infrastructure) with a permissive SCCs, there is nothing particular for you to configure. Here, a permissive SCCs refers to one where the strategy for `RunAsUser` is defined as `RunAsAny` (including root).
+## Deploying Camunda 8 in OpenShift
 
-#### Non-root SCCs
+Depending on your OpenShift cluster's Security Context Constraints (SCCs) configuration, the deployment process may vary.
 
-If you wish to deploy Camunda 8 but restrict the applications from running as root (e.g. the `nonroot` built-in SCCs), you will need to configure each pod and container to run as a non-root user. For example, when deploying Zeebe using a stateful set, you would add the following, replacing `1000` with the user ID you wish to use:
+<Tabs>
+  <TabItem value="w-scc" label="Security Context Constraints (SCCs)" default>
 
-```yaml
-spec:
-  template:
-    spec:
-      securityContext:
-        runAsUser: 1000
-      containers:
-        securityContext:
-          runAsUser: 1000
+### With restrictive SCCs
+
+By default, OpenShift employs more restrictive SCCs. The Helm chart must assign `null` to the user running all components and dependencies. Due to a [null bug](https://github.com/helm/helm/issues/9136) [in Helm](https://github.com/helm/helm/issues/12490), this operation must be executed using [a post-renderer](https://helm.sh/docs/topics/advanced/#post-rendering).
+
+To deploy Camunda 8 on OpenShift, please follow these installation steps:
+
+1. Install [Helm and other CLI tools](/self-managed/setup/install.md#prerequisites).
+2. Ensure that `bash` and `sed` are available locally, as they are necessary for the [post-rendering process to patch the values of OpenShift](https://github.com/camunda/camunda-platform-helm/blob/main/charts/camunda-platform/openshift/patch.sh).
```

Comment:
should we be more specific on sed?
The one on mac can be very different to whatever one find on e.g. debian.

Reply:
haven't tried it out yet whether it works the same on both mac and linux as the mac sed can be limited

Reply:
just tried it out, with mac sed it doesn't do anything not even throw an error and the values were still present.
On debian / Linux whatever gnu sed, worked correctly.

Reply:
From my perspective https://github.com/camunda/camunda-platform-helm/pull/1970 released would be the right approach.